default.yaml

```diff
@@ -167,6 +167,11 @@ onChainWallet:
   scanDepthOutgoing: 2
   scanDepthChannelUpdate: 8
 
+loop:
+  maxOutboundLiquidityBalance: 2835916 # 0.03 ($600)
```

Comment:
probably while you are testing but please remove unnecessary comments or please add useful comments